setup.sh

```diff
@@ -76,12 +76,19 @@ fi
 echo "Downloading ${URL}..."
 
 # Download installation zip.
-curl --connect-timeout 15 --retry 5 "$URL" > "${HOME}/dartsdk.zip"
-unzip -o "${HOME}/dartsdk.zip" -d "${RUNNER_TOOL_CACHE}" > /dev/null
+curl --connect-timeout 15 --retry 5 --retry-all-errors "$URL" \
```

Comment:
We need to feature test for the retry-all-errors here, otherwise we are incompatible with some older image versions.